### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#483] [Esami & Esoneri M-Z]
### Traccia

<img width="941" alt="image" src="https://github.com/user-attachments/assets/7c20755a-17cb-4447-b679-d482f6c8c30c">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/d31366e3-aba6-47fa-bbc7-b8c43ef4e7b9)
